capstone/src/main/java/com/google/sps/data/AssociationData.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+/** Data for top negative and positive associations  */
+public final class AssociationData {
+
+  private ArrayList<String> positive;
+  private ArrayList<String> negative;
+	      
+  public AssociationData(ArrayList<String> positiveAssociations, ArrayList<String> negativeAssociations) {
```

Comment:
Ensure that you keep the column limit under 100 characters. Java has a specific guideline that recommends keeping column character limits below 100
https://google.github.io/styleguide/javaguide.html#s4.4-column-limit